Makefile

```diff
@@ -366,6 +387,10 @@ loop: mod core
 	@$(CC) -c -o $(PREFIX)/loop_ping.o test/loop_ping.c $(CFLAGS) -ggdb
 
 units: mod core lib $(UNITS_OBJ) $(MOD_OBJ)
+	@if [ $(UNITS) -eq 0 ]; then \
```

Comment:
makefiles don't support that behaviour. If you can figure out a way to do it more cleanly, I'm all ears.